packages/driver/patches/jquery+3.4.1.dev.patch

```diff
@@ -89,7 +89,7 @@ index 773ad95..84ca2f6 100644
 +	// IE/Edge sometimes throw a "Permission denied" error when strict-comparing
 +	// two documents; shallow comparisons work.
 +	// eslint-disable-next-line eqeqeq
-+	if ( preferredDoc != document &&
++	if ( docElem.msMatchesSelector && preferredDoc != document &&
```

Comment:
is this still relevant? Mostly curious how you tested this.

Reply:
This is basically a cherry-picked patch from jQuery 3.7.1. when manually testing this in chrome, I was still receiving unload errors coming from jquery without this change